src/components/va-alert/test/va-alert.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { VaAlert } from '../va-alert';
+
+describe('va-alert', () => {
+
+  it('fires an custom event when the component is loaded', async () => {
+    const alert = new VaAlert();
+    alert.vaComponentDidLoad = {emit:jest.fn()};
+    alert.componentDidLoad();
+    expect(alert.vaComponentDidLoad.emit).toHaveBeenCalled();
+  });
```

Comment:
:thinking: I'd prefer to see this test what happens when the component actually renders. Can you update this to use `newE2EPage` so the component renders in an actual browser? You can check out what we did for another component for [testing event emitting](https://github.com/department-of-veterans-affairs/component-library/blob/web-components/src/components/va-checkbox/test/va-checkbox.e2e.ts#L167-L178).